README.adoc

```diff
@@ -148,83 +155,84 @@ Write a simple `Consumer` class that consumes the artist JSON using both JSON-P
 include::finish/src/main/java/io/openliberty/guides/consumingrest/Consumer.java[tags=!comment;!class]
 ----
 
-=== Processing JSON using JSON-P
-
-Write the method for consuming the REST service using JSON-P in the `Consumer` class:
-
-[source, java, indent=0]
-----
-include::finish/src/main/java/io/openliberty/guides/consumingrest/Consumer.java[tags=consumeWithJsonp]
-----
+=== Processing JSON using JSON-B
 
-This method makes a `GET` request to the running artist service and retrieves the JSON.
+JSON-B is a Java API that is used to serialize Java objects to JSON messages and vice versa.
 
-Write the `collectArtists` and `collectAlbums` helpers. These methods will parse that JSON and
-collect its objects into individual POJOs:
+To include the JSON-B provider in your project, add the following dependency to your `pom.xml`,
+which has been done for you:
 
-[source, java, indent=0]
+[source, xml]
```

Comment:
Can you add `role="no_copy` to distinguish this listing block since it doesn't need to be copied.